api/src/pcapi/core/offerers/api.py

```diff
@@ -2171,7 +2171,7 @@ def set_accessibility_infos_from_provider_id(venue: models.Venue) -> None:
 
 def count_permanent_venues_with_accessibility_provider() -> int:
     return (
-        offerers_models.Venue.query.outer(offerers_models.AccessibilityProvider)
+        offerers_models.Venue.query.join(offerers_models.AccessibilityProvider)
         .options(
```

Comment:
Même si ce n'est pas l'objet principal de cette PR : 

Tu peux en profiter pour retirer le `load_only`, car je ne saisis pas le besoin de charger des colonnes quand on fait juste un `count()` . On n'en a pas besoin pour le `filter()` en-dessous, le `join` suffit`.

Également, tu peux remplacer les `== True` et `== False` par `.is_(True)` et `.is_(False)` par homogénéité du code (et dans la fonction en-dessous).